### ISSUE TITLE
Ctrl + click

### ISSUE THREAD

**Issue by meramjanam** [microsoft/vscode#117330]
Issue Type: <b>Feature Request</b>

Ctrl + click is not working
Go to definition is not workin

VS Code version: Code 1.53.2 (622cb03f7e070a9670c94bae1a45d78d7181fbd4, 2021-02-11T11:48:04.245Z)
OS version: Windows_NT x64 10.0.19042

**Comment by IllusionMH** [comment]
/needsMoreInfo

Language and minimal code sample required.

**Comment by rzhao271** [comment]
Yep. In general, does Ctrl+click work for you on Insiders? Alternatively, attach a screencap showing how it is not working. You can record a screencap using https://gifcap.dev.

**Comment by meramjanam** [comment]
Please find recording
![recording](https://user-images.githubusercontent.com/43846363/108972917-cf09ab00-76a9-11eb-97a9-95dff4dfd4c9.gif)

**Comment by hahmadia** [comment]
Please fix! I'm experiencing this too all of a sudden and it's a pain in the butt cause I relied on that a bit...

**Comment by davidpoza** [comment]
Me too.
It works when vscode starts, but after some time the problem appears.

Vscode 1.55.2
PopOs 20.10.
